sklearn/preprocessing/_encoders.py

```diff
@@ -603,8 +605,6 @@ class OrdinalEncoder(_BaseEncoder):
 
     Read more in the :ref:`User Guide <preprocessing_categorical_features>`.
 
-    .. versionchanged:: 0.20.1
```

Comment:
We should note when OrdinalEncoder was added but maybe that's intentionality left for another pull request?